songs_alt.php

```diff
@@ -0,0 +1,40 @@
+<?php 
+    require 'graeme_grooves_mysqli.php';
```

Comment:
This can be viewed without logging in. N.b. cannot be accessed unless manually navigated to